qa/acceptance/spec/shared_examples/cli/logstash-plugin/install.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+require "logstash/devutils/rake"
```

Comment:
we should remove this from here, test can we run with rake and with directly the rspec tool, for example to run only 1 example as requested #5228